.travis.yml

```diff
@@ -1,11 +1,34 @@
 language: node_js
-node_js:
-  - 8
+
+before_install:
+  # Export the C++11 compiler.
+  - if [[ $TRAVIS_OS_NAME == "linux" ]]; then export CXX=g++-4.8; fi
```

Comment:
So what is this for and why do we need a C++ compiler?

Reply:
My mistake. I forgot to remove it.  Fixed